frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs

```diff
@@ -16,12 +17,48 @@
    [app.main.refs :as refs]
    [app.main.store :as st]
    [app.main.ui.components.numeric-input :refer [numeric-input*]]
+   [app.main.ui.components.radio-buttons :refer [radio-button radio-buttons]]
    [app.main.ui.components.select :refer [select]]
+   [app.main.ui.components.title-bar :refer [title-bar]]
+   [app.main.ui.context :as ctx]
    [app.main.ui.icons :as i]
    [app.util.dom :as dom]
    [cuerdas.core :as str]
    [rumext.v2 :as mf]))
 
+(defn dir-icons-refactor
+  [val]
+  (case val
+    :row            i/grid-row-refactor
+    :row-reverse    i/row-reverse-refactor
+    :column         i/column-refactor
+    :column-reverse i/column-reverse-refactor))
+
+(mf/defc direction-btn
+  [{:keys [dir saved-dir set-direction icon?] :as props}]
```

Comment:
I suggest you to change `set-direction` to `on-change`.
Prefer using standard callback naming convention, that makes clear visual distinction which attrs are functions and which ones are normal values.